graph/telemetry/istio/appender/aggregate_node.go

```diff
@@ -71,7 +72,7 @@ func (a AggregateNodeAppender) AppendGraph(trafficMap graph.TrafficMap, globalIn
 }
 
 func (a AggregateNodeAppender) appendGraph(trafficMap graph.TrafficMap, namespace string, client *prometheus.Client, conf *config.Config) {
-	log.Tracef("Resolving request aggregates for namespace=[%s], aggregate=[%s]", namespace, a.Aggregate)
+	a.log.Tracef("Resolving request aggregates for namespace=[%s], aggregate=[%s]", namespace, a.Aggregate)
```

Comment:
appenders I think would also benefit from getting the logger from the context so the request id can be included. Then we can log "appenders took x seconds to complete" along with the request id but unfortunately we're not passing in the context into the appenders. Maybe the context is saved somewhere else the appenders can access?